.github/workflows/publish.yaml

```diff
@@ -48,39 +47,51 @@ jobs:
           inv generateapidocs
           inv checkchanges --action="run inv generateapidocs"
 
-  build-and-push:
-    needs: unit-tests
+  publish-images:
     runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: true
+      matrix:
+        image: [speaker, controller, mirror-server, configmaptocrs]
```

Comment:
We haven't published mirror-server until now, and as @fedepaol said we might be able to remove it completely (in a different pr ofc)

Reply:
Ok, can remove